verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -257,26 +198,11 @@ class KytheFactsExtractor {
 
   // Keeps track of scopes and definitions inside the scopes of ancestors as
   // the visitor traverses the facts tree.
```

Comment:
This would be a good place to explain with some examples what you mean by vertical and horizontal scope.